de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java

```diff
@@ -258,21 +165,32 @@ public HandlerRegistration addAppSelectionChangedEventHandler(AppSelectionChange
     }
 
     @Override
-    public void onAppInfoSelected(AppInfoSelectedEvent event) {
+    public void onAppInfoSelected(Splittable app) {
+        AppInfoSelectedEvent event = new AppInfoSelectedEvent(splittableToApp(app));
         fireEvent(event);
     }
 
+    App splittableToApp(Splittable app) {
+        return AutoBeanCodex.decode(factory, App.class, app).as();
```

Comment:
This splittable to app and app to splittable conversion can be eliminated once other parts of apps window is `react`ified